apps/builder/app/canvas/instance-hovering.ts

```diff
@@ -65,7 +65,7 @@ export const subscribeInstanceHovering = () => {
       instanceId: instance.id,
       rect: getAllElementsBoundingBox(element),
     });
-  }, 50);
+  }, 0);
```

Comment:
do we still need the debounce at all?

Reply:
Dont need just to show that it feels better